doc/vulnmeet-presentation.md

```diff
@@ -0,0 +1,242 @@
+
+# TryTLS [![CircleCI](https://circleci.com/gh/ouspg/trytls.svg?style=shield)](https://circleci.com/gh/ouspg/trytls) [![Build status](https://ci.appveyor.com/api/projects/status/91p39fn87pbiy1gs?svg=true)](https://ci.appveyor.com/project/jviide/trytls)
+
+Does *your* TLS/SSL library check certificates properly?
+Broken certificate checks seems to be an overlooked issue.
+Handling certificates is surprisingly complex, and calls for extra attention.
+
+TryTLS is a tool for the software and library developers, vulnerability
+researchers, and end-users, who want to use TLS safely.
+
+We hope to help you to test certificate handling easily. We support
+systematic and readily planned tests and try make integrating your
+favourite language and library easy.
+
+We are completely open source, feel free to contribute!
+
+---
+
+# What TryTLS Is Not
+
+* We do not address possible client certificate check problems in server code
+* We do not do or require a man-in-the-middle tools
+* We do not support smart TVs, IoT toasters and other such devices that
+  can't run any of the runners
+
+---
+
+# How Does It Work
+
+![Architecture](https://raw.githubusercontent.com/ouspg/trytls/master/doc/concept-pic.png)
+
+* **Backends (servers)** provide TLS/SSL tests for stubs (clients).
+* **Stubs** attempt to establish secure TLS/SSL connections to backends.
+* **Runners** run the stubs against backends and provide the results of the tests.
+
+---
+
+# Runners
+
+* trytls (official)
+* bashtls + simplerunner (unofficial)
+
+Installation
+
+```sh
+pip install trytls
+```
+
+Usage
+
+```sh
+$ git clone https://github.com/ouspg/trytls.git
+$ trytls https python trytls/stubs/python-urllib2/run.py
+platform: OS X 10.11.5
+runner: trytls 0.2.0 (CPython 2.7.10, OpenSSL 0.9.8zh)
+stub: python 'run.py'
+ PASS support for TLS server name indication (SNI) [accept badssl.com:443]
+ PASS expired certificate [reject expired.badssl.com:443]
+ PASS wrong hostname in certificate [reject wrong.host.badssl.com:443]
+  ...
+```
+
+---
+
+# Stubs
+
+Stubs attempt to establish a **secure** connection to the given
+service(host + port) and catch possible errors and exceptions to
+determine if the connection was successful.
+
+The last string the stub prints is the verdict (UNSUPPORTED,
+ACCEPT etc.). If the stub provides additional context such as
```

Comment:
replace "etc." with "REJECT" since that's the third verdict and there's no more verdicts. Also, maybe reorder verdicts here as ACCEPT, REJECT, UNSUPPORTED.